pyscriptjs/src/pyscript.py

```diff
@@ -385,32 +380,37 @@ def revert(self):
         console.log("----> reverted")
         self._out = self._prev
 
-    def write(self, txt):
-        console.log("writing to", self._out, txt, self._append)
+    def write(self, value):
+        console.log("writing to", self._out, value, self._append)
         if self._out:
-            pyscript.write(self._out, txt, append=self._append)
+            Element(self._out).write(value, self._append)
+
         if self.output_to_console:
-            console.log(self._out, txt)
+            console.log(self._out, value)
 
 
 class OutputManager:
     def __init__(self, out=None, err=None, output_to_console=True, append=True):
         sys.stdout = self._out_manager = OutputCtxManager(
-            out, output_to_console, append
+            out=out, output_to_console=output_to_console, append=append
         )
         sys.stderr = self._err_manager = OutputCtxManager(
-            err, output_to_console, append
+            out=err, output_to_console=output_to_console, append=append
         )
         self.output_to_console = output_to_console
         self._append = append
 
     def change(self, out=None, err=None, output_to_console=True, append=True):
-        self._out_manager.change(out, output_to_console, append)
+        self._out_manager.change(
+            out=out, output_to_console=output_to_console, append=append
+        )
         sys.stdout = self._out_manager
-        self._err_manager.change(err, output_to_console, append)
+        self._err_manager.change(
+            err=err, output_to_console=output_to_console, append=append
```

Comment:
this should be `out=err` not `err=err` . I'm testing locally and can fix it, just commenting for posterity :)